app/controllers/api/coordinates_controller.rb

```diff
@@ -1,6 +1,6 @@
 require 'geocoding'
 class Api::CoordinatesController < Api::ApplicationController
-  before_action :verify_json_request, only: ['show']
+  before_action :verify_json_request, only: %w[show]
```

Comment:
Is `%i[:show]` also valid? We do that in a few places, so it might nice to pick one and be consistent